### ISSUE TITLE
BUG - Menu Sobre

### ISSUE THREAD

**Issue by Danny-s07** [dev-pira/site#203]
**Describe the bug**

Problema no menu 

**To Reproduce**

1-Problema está no menu do desktop e também afeta o mobile
2- No Sobre , esta apenas recarregando a pagina

**Expected behavior**

O sobre era para estar  ancorado ou indo para alguma pagina do sobre, isso não ocorre, está dando refresh na pagina e permanecendo no endereço inicial. Já os outros itens do menu estão sendo redirecionados corretamente

**Screenshots**

Opções de Como deveria estar :
Menu ancorado 
![image](https://github.com/dev-pira/site/assets/97040972/d13fdd98-bbb0-48f4-b84b-6b637113c473)

Menu indo para uma pagina especifica
![image](https://github.com/dev-pira/site/assets/97040972/2323a2d3-390c-4407-b65a-a698638c3855)

**Desktop (please complete the following information):**

- OS: Windows 
- Browser  : Microsoft Edge , Opera Gx e Google Chrome
- Version : M Edg -119 final 93  ; Opera GX - 104 final 85 ; Google Chrome 119 final 200

**Smartphone (please complete the following information):**

- Device: Android  Motorola 
- OS: Android 9
- Browser : google chrome
- Version : 120 final 44

**Additional context**
Mesmo que ainda não tenha a pagina  ou que esteja em desenvolvimento(produção) e ainda não subiu, deixa ancorado pelo menos. Já não vai ficar quebrado desse jeito....

**Comment by murilobeltrame** [comment]
Fala @Danny-s07 , não encontrei o menu "Sobre".
Era sobre isso mesmo ?

**Comment by Danny-s07** [comment]
> Fala @Danny-s07 , não encontrei o menu "Sobre". Era sobre isso mesmo ?

@murilobeltrame  sim, isso esta acontecendo quando vai no menu e clica na palavra sobre la no site que ja esta no ar . Não sei se um video talvez facilite pra vocês ? e também não sei se consigo enviar

**Comment by murilobeltrame** [comment]
Agora peguei, eu estava cego aqui ... kkkk
O "Sobre" tem o comportamento programado para ir pra página inicial ... um comportamento equivalente a clicar no Logo, 
Quando você já estiver na pagina inicial, a página realmente só vai recarregar ... se você navegar para a pagina de eventos ou de vagas e tentar clicar em sobre, então ela deve voltar para a página inicial.

**Comment by Danny-s07** [comment]
> Agora peguei, eu estava cego aqui ... kkkk O "Sobre" tem o comportamento programado para ir pra página inicial ... um comportamento equivalente a clicar no Logo, Quando você já estiver na pagina inicial, a página realmente só vai recarregar ... se você navegar para a pagina de eventos ou de vagas e tentar clicar em sobre, então ela deve voltar para a página inicial.

Isso , o bug está ai kssksk
